docs/en/02_Developer_Guides/14_Files/05_File_Migration.md

```diff
@@ -63,20 +66,22 @@ are incompatible with core file security.
 
 ## Support existing paths
 
-Because the filesystem now uses the sha1 of file contents in order to version multiple versions under the same
+Because the filesystem now uses the hash of file contents in order to version multiple versions under the same
 filename, the default storage paths in 4.0 will not be the same as in 3.
 
-Although it is not recommended, it is possible to configure the backend to omit this SHA1 url segment,
+Although it is not recommended, it is possible to configure the backend to omit this hash url segment,
 meaning that file paths and urls will not be modified during the upgrade.
-
-This is done by setting this config:
+This configuration needs to be chosen before starting the file migration,
```

Comment:
Do you think it's worth bolding this part?

Reply:
Not particularly, you need to read the whole section anyway